tests/models/conditional_detr/test_feature_extraction_conditional_detr.py

```diff
@@ -303,7 +303,6 @@ def test_call_pytorch_with_coco_panoptic_annotations(self):
         masks_path = pathlib.Path("./tests/fixtures/tests_samples/COCO/coco_panoptic")
 
         # encode them
-        # TODO replace by .from_pretrained microsoft/conditional-detr-resnet-50-panoptic
```

Comment:
OUt of curiosity, why are we not doing this anymore?

Reply:
There's no Conditional DETR checkpoint on the hub for panoptic segmentation. Hence I'm just instantiating a feature extractor locally.